src/test/storm-gamebased-ar/abstraction/PrismMenuGameTest.cpp

```diff
@@ -575,145 +356,22 @@ TEST(PrismMenuGame, TwoDiceAbstractionTest_Sylvan) {
 
     std::shared_ptr<storm::utility::solver::SmtSolverFactory> smtSolverFactory = std::make_shared<storm::utility::solver::MathsatSmtSolverFactory>();
 
-    storm::gbar::abstraction::prism::PrismMenuGameAbstractor<storm::dd::DdType::Sylvan, double> abstractor(program, smtSolverFactory);
-    storm::gbar::abstraction::MenuGameRefiner<storm::dd::DdType::Sylvan, double> refiner(abstractor, smtSolverFactory->create(manager));
+    storm::gbar::abstraction::prism::PrismMenuGameAbstractor<DdType, double> abstractor(program, smtSolverFactory);
+    storm::gbar::abstraction::MenuGameRefiner<DdType, double> refiner(abstractor, smtSolverFactory->create(manager));
     refiner.refine(initialPredicates);
 
-    storm::gbar::abstraction::MenuGame<storm::dd::DdType::Sylvan, double> game = abstractor.abstract();
+    ASSERT_NO_THROW(refiner.refine({manager.getVariableExpression("d1") + manager.getVariableExpression("d2") == manager.integer(7)}));
 
-    EXPECT_EQ(90ull, game.getNumberOfTransitions());
```

Comment:
are these missing?

Reply:
The diff is comparing two different functions. The assertions are in lines 554 (old) or 335 (new).